mysys_ssl/my_default.cc

```diff
@@ -668,6 +668,7 @@ int my_load_defaults(const char *conf_file, const char **groups,
                                      handle_default_option, (void *) &ctx,
                                      dirs)))
   {
+    delete_dynamic(&args);
```

Comment:
Add another "delete_dynamic(&args);" at the code block above (lines 656 - 662)

>    if ((error= my_search_option_files(conf_file, argc, argv,
                                      &args_used, handle_default_option,
                                      (void *) &ctx, dirs)))
   {
     free_root(&alloc,MYF(0));
     DBUG_RETURN(error);
   }